VnE/Source/Extensions/ChemPack/Db_viewer.py

```
@@ -605,6 +604,12 @@ def importRefs(self):
             data = [x for x in file.read().split('\n') if x]
             self.list_model.iterPopulate(((x, 'refcode', 'cryst', 'refcode', {}) for x in data))
 
+    def downloadRefs(self):
+        filename, _ = QtWidgets.QFileDialog.getOpenFileName(filter='*.txt')
+        if filename:
+            file = open(filename, 'r')
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)